src/core/distributions/Distribution.h

```diff
@@ -53,7 +53,7 @@ namespace RevBayesCore {
         void                                                    restore(const DagNode *restorer);
         virtual void                                            setMcmcMode(bool tf);                                                               //!< Change the likelihood computation to or from MCMC mode.
         void                                                    swapParameter(const DagNode *oldP, const DagNode *newP);                            //!< Exchange the parameter
-        void                                                    touch(const DagNode *toucher, bool touchAll);
+        virtual void                                            touch(const DagNode *toucher, bool touchAll);
```

Comment:
I think maybe you are supposed to leave this as `void`, and override `touchSpecialization( )` below, which is already virtual.

If you look in `Distribution.cpp`, you'll see that what `touch( )` does is to call `touchSpecialization( )`.

Reply:
Yes, this makes a lot of sense. Done.